ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -411,23 +407,11 @@ export const ConceptHomepage = withCurrentWorkspace()(
       return selectedConceptDomainMap[selectedDomain.domain].length;
     }
 
-    selectedQuestion(selectedQues) {
```

Comment:
Does the api need a call the get survey answers? Or is that obsolete now?

Reply:
We've removed the answers since they're not selectable. It's possible we may need them in the future though